.travis.yml

```diff
@@ -1,14 +1,40 @@
+---
+
+sudo: required
+
+# N/B: host go env. not actually used, see .run_ci_tests.sh
 language: go
 go:
-    - 1.9
-    - 1.8
-    - 1.7
-dist: trusty
-sudo: required
+    - master
+
+services:
+  - docker
+
+env:
+    global:
+        # In Travis $PWD == $TRAVIS_BUILD_DIR == subdir of $HOME, this makes that happen
+        - REPO_VOL=$HOME
+        - TRAVIS_ENV="-e TRAVIS=$TRAVIS
+                      -e CI=$CI
+                      -e TRAVIS_COMMIT=$TRAVIS_COMMIT
+                      -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG
+                      -e TRAVIS_PULL_REQUEST=$TRAVIS_PULL_REQUEST
+                      -e TRAVIS_PULL_REQUEST_SHA=$TRAVIS_PULL_REQUEST_SHA
+                      -e TRAVIS_PULL_REQUEST_SLUG=$TRAVIS_PULL_REQUEST_SLUG
+                      -e TRAVIS_BRANCH=$TRAVIS_BRANCH
+                      -e TRAVIS_JOB_ID=$TRAVIS_JOB_ID
+                      -e TRAVIS_BUILD_DIR=$TRAVIS_BUILD_DIR"
+    matrix:
+        - GO_VERSION="stable"
+        - GO_VERSION="1.9"
+        - GO_VERSION="1.8"
+        - GO_VERSION="1.7"
+
 before_install:
     - sudo apt-get -qq update
-    - sudo apt-get -qq install btrfs-tools libdevmapper-dev
```

Comment:
I think we are going to need these inside of the dockerfile?

Reply:
I left those in ``.spc_ci_test.sh`` as ``$DEBS``, thinking y'all wouldn't want to to touch dockerfiles.  The one for ``docker.io/cevich/travis_ubuntu:latest``, I tried to keep close to what Travis does by default  (package-wise).  The only non-packaged item is go, which gets installed before the test by that ``gimme`` tool - as travis does it.

Speaking of which...I set up ``travis_ubuntu`` to follow ``docker.io/ubuntu:latest`` automatically, which could cause surprises (if it changes in the middle of a test).  I don't see it as a likely problem though, and it can be "fixed" by just re-running the tests.  However, I'm happy to use a different arrangement or image if y'all prefer.